src/components/site/PageHeader.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+const PageHeader = (props) => (
+	<div className="dg_page-header">
+		<div className="dg_page-header__box ">
```

Comment:
The more meaningful comment is below, but this can be changed to just `className="row"`

Reply:
done